File: python/7_kyu/middle_character.py
# https://www.codewars.com/kata/56747fd5cb988479af000028/train/javascript

import math

def get_middle(string):
    index = math.floor(len(string) / 2)
    if len(string) % 2 == 0:
        return f"{string[index - 1]}{string[index]}"
    return string[index]


print(get_middle("test"))

File: javascript/7_kyu/middle_character.js
// https://www.codewars.com/kata/56747fd5cb988479af000028/train/javascript

function getMiddle(string) {
  const middleIndex = string.length / 2;
  if (string.length % 2 == 0)
    return `${string.charAt(middleIndex - 1)}${string.charAt(middleIndex)}`;
  return string.charAt(middleIndex);
}

console.log(getMiddle("test"));
